src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -48,7 +48,8 @@ public string CookieName
 
         /// <summary>
         /// The path set on the cookie. If it's <c>null</c>, the "path" attribute on the cookie is set to current
-        /// request's <see cref="HttpRequest.PathBase"/> value.
+        /// request's <see cref="HttpRequest.PathBase"/> value. If the value of <see cref="HttpRequest.PathBase"/> is
```

Comment:
Do this: https://github.com/aspnet/Mvc/pull/5521#discussion_r87125822

Then :shipit: